src/AuthorStrategyFactory.ts

```
@@ -0,0 +1,25 @@
+      return new DefaultAuthorStrategy();
+    }
+    switch (platform) {
+      case PlatformType.Jekyll:
+        return new JekyllAuthorStrategy();
+      case PlatformType.Docusaurus:
+        return new DocusaurusAuthorStrategy();
+      default:
+        return new DefaultAuthorStrategy();
+    }
+  }
+}
```

Biome

[error] Avoid classes that contain only static members.

Prefer using simple functions instead of classes with only static members.

(lint/complexity/noStaticOnlyClass)